Traceback (most recent call last):
  File "/home/rgg2706/Multimodal-Sentiment-Analysis/Models/MOASC4/run_project.py", line 387, in <module>
    run_sentiment_classifier(multimodal_train_loader, multimodal_val_loader, multimodal_test_loader, params, device)
  File "/home/rgg2706/Multimodal-Sentiment-Analysis/Models/MOASC4/run_project.py", line 302, in run_sentiment_classifier
    train(train_loader, validation_loader, test_loader, params, device)
  File "/home/rgg2706/Multimodal-Sentiment-Analysis/Models/MOASC4/run_project.py", line 236, in train
    logits = model(sequences, attn_masks, images)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/.autofs/tools/spack/var/spack/environments/default-nlp-x86_64-24072401/.spack-env/._view/wtxwc3mlkmzy7fbaxlum2674jarpitc2/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/.autofs/tools/spack/var/spack/environments/default-nlp-x86_64-24072401/.spack-env/._view/wtxwc3mlkmzy7fbaxlum2674jarpitc2/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rgg2706/Multimodal-Sentiment-Analysis/Models/MOASC4/SentimentClassifier.py", line 153, in forward
    d = list(cont_reps.size())[2]
             ^^^^^^^^^^^^^^
AttributeError: 'BaseModelOutputWithPoolingAndCrossAttentions' object has no attribute 'size'
